src/utils/utils.jl

```diff
@@ -367,7 +367,7 @@ function get_initial_times(
     if count == 0
         return []
     elseif count == 1
-        @assert interval == Dates.Second(0)
+        @assert interval == Dates.Second(0) "interval=$interval"
```

Comment:
I have never seen this syntax of @assert and then string. Does this work?

Reply:
yes

Reply:
That's cool. It will make it far easier to understand

Reply:
What we should do is make a macro that does this for you automatically.